src/UseCases/GetInTouch/GetInTouchUseCase.php

```diff
@@ -24,17 +25,17 @@ public function __construct( GetInTouchValidator $validator, Messenger $messenge
 		$this->messenger = $messenger;
 	}
 
-	public function processContact( GetInTouchRequest $request ): string {
+	public function processContact( GetInTouchRequest $request ): ValidationResponse {
 		$this->request = $request;
 
 		if ( !$this->validator->validate( $request ) ) {
-			return 'validation failed';
+			return ValidationResponse::newFailureResponse( $this->validator->getConstraintViolations() );
 		}
 
 		try {
 			$this->forwardContactRequest();
 			$this->confirmToUser();
-			return 'request successful';
+			return ValidationResponse::newSuccessResponse();
 		} catch ( \RuntimeException $e ) {
 			return 'mail transmission failed';
```

Comment:
You're right, the way the code got changed now will cause it to error out on error. Not sure about not catching the exception. Generally not nice to have such service specific stuff bubble up. Isn't returning a failure response model here very easy?